superset-frontend/src/dashboard/stylesheets/components/chart.less

```diff
@@ -23,7 +23,7 @@
   background-color: @lightest;
   position: relative;
   padding: 16px;
-  overflow-y: auto;
+  overflow-y: visible;
```

Comment:
Unfortunately this change will affect `Region Filter`:
`overflow-y: visible` (now):
<img width="150" alt="Screenshot 2021-01-04 at 16 15 00" src="https://user-images.githubusercontent.com/2536609/103549981-83583380-4ea8-11eb-8416-998369f2029c.png">

and  `overflow-y: auto` (was):
<img width="141" alt="Screenshot 2021-01-04 at 16 14 46" src="https://user-images.githubusercontent.com/2536609/103550008-8e12c880-4ea8-11eb-98e9-5c9c485fb26f.png">

I changed scroll solution back [here](https://github.com/apache/incubator-superset/issues/11419) but I believe there must be some better solution here.

Reply:
![Kapture 2021-01-04 at 19 20 42](https://user-images.githubusercontent.com/8277264/103561392-1a7db500-4ec2-11eb-9835-37e38c0593d0.gif)
@kkucharc @junlincc

Reply:
@kkucharc I just pulled this, and confirm that what I see looks like @maloun96 's video above. While the scrolling is... weird, it seems comparable to what was there before. I think we're OK, though I hope to find a better design solution to tiny charts/filters like this, as the scrolling is goofy. 

I think we're OK here, but let me know if you see any remaining issue.